concepts/observability-strategy/configurable-tracing/04-tracing-api.md

```diff
@@ -19,18 +19,37 @@ apiVersion: telemetry.kyma-project.io/v1alpha1
 metadata:
   name: myPipeline
 spec:
+  input:
+    tags: # Filter based on custom tags (Is there any use-case or would we just produce incomplete traces?)
```

Comment:
will for sure produce incomplete traces as istio spans and serverless spans have different tags...